client/browser/src/shared/cli/search.ts

```diff
@@ -101,6 +107,8 @@ export class SearchCommand {
                 if (isDefined(settings) && isNot<ErrorLike | Settings, ErrorLike>(isErrorLike)(settings)) {
                     this.defaultPatternType =
                         (settings['search.defaultPatternType'] as SearchPatternType) || this.defaultPatternType
+                    this.defaultCaseSensitive =
+                        (settings['search.defaultCaseSesntive'] as boolean) || this.defaultCaseSensitive
```

Comment:
```suggestion
                        (settings['search.defaultCaseSensitive'] as boolean) || this.defaultCaseSensitive
```

Typo was causing omnibox search via extension to not apply the setting :)

Reply:
![leak](https://user-images.githubusercontent.com/888624/98043327-473e1f00-1de2-11eb-9e92-afe600a635c3.gif)

thank you!